src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -1,32 +1,27 @@
-// package seedu.address.logic.parser;
-//
-// import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-//
-// import java.util.Arrays;
-//
-// import seedu.address.logic.parser.exceptions.ParseException;
-// import seedu.address.model.person.NameContainsKeywordsPredicate;
-//
-// /**
-//  * Parses input arguments and creates a new FindCommand object
-//  */
-// public class FindCommandParser implements Parser<FindCommand> {
-//
-//     /**
-//      * Parses the given {@code String} of arguments in the context of the FindCommand
-//      * and returns a FindCommand object for execution.
-//      * @throws ParseException if the user input does not conform the expected format
-//      */
-//     public FindCommand parse(String args) throws ParseException {
-//         String trimmedArgs = args.trim();
-//         if (trimmedArgs.isEmpty()) {
-//             throw new ParseException(
-//                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
-//         }
-//
-//         String[] nameKeywords = trimmedArgs.split("\\s+");
-//
-//         return new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
-//     }
-//
-// }
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.recipe.RecipeContainsKeywordPredicate;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class FindCommandParser implements Parser<FindCommand> {
+
+    @Override
+    public FindCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        }
```

Comment:
This check might not be necessary(?) i think AB3 commands don't check for this either (argument param shouldn't be empty since I THINK it's alr checked in the Addressbookparser/Recipebookparser class)